wordpress-theme/skyyrose-flagship/assets/css/luxury-theme.css

```
@@ -244,21 +286,399 @@ label {
+	text-decoration: none;
+}
+
+.site-branding .site-description {
+	display: none;
+}
+
+/* ==================================================
+   BRAND LOGOS — Primary SR Monogram
+   ================================================== */
+
+@keyframes logoShimmer {
```

Stylelint (17.3.0)

[error] Expected keyframe name "logoShimmer" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)